src/utils/reportHelpers.ts

```diff
@@ -690,29 +707,43 @@ export const convertToReportRows = (rows: IPlanningRow[], reportType: ReportType
             projects: c.projectRows.length ? convertToConstructionReportProjects(c.projectRows, divisions) : [],
             type: getConstructionRowType(c.type, c.name.toLowerCase()) as ReportTableRowType,
           }
+
+          // 'Esirakentaminen' is old budget item and it should be hidden on the report.
+          // We rename it as '' that will be used later to skip row if the name is empty.
+          if (c.name === 'Esirakentaminen' && c.path === '8 01 Kiinteä omaisuus/Esirakentaminen'){
+            convertedClass.name = '';
+          }
+
```

Comment:
I'm seeing this Esirakentaminen row on the report
<img width="143" alt="image" src="https://github.com/City-of-Helsinki/infraohjelmointi-ui/assets/59826954/cd2d9121-73bf-4339-b97d-23c406a3ba29">

Reply:
We investigated this with PMs, and the environment had old class numbers (e.g. "801" without spaces).